bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
But you are missing the point. If you use items() now, then the actual 2to3 converter which we will have to run is going to make them into [items()] and then we'll have to go through and remove them. If you use iteritems() now, you will be *doing* what python3 does with items() and the conversion will generate the right code.